README.md

```diff
@@ -42,6 +42,8 @@ Collection of examples that solve specific problems using small pieces of code.
 
 ### FastAPI Examples
 
+- [CLI Base Example](./examples/cli-base/README.md)
+- [CLI with Typer + Rich Example](./examples/cli-typer-base/README.md)
```

Comment:
This is not a fast api example.. can you add a new section above called CLI Tools?

Reply:
Sure!